app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
+  margin-bottom: $base-spacing * 4;
```

Comment:
@tysongach the HTML looks like:

```
<div class="pagination--with-total">
  <nav class='pagination'>
    ...
  </nav>
  <div class="pagination__total">
    ...
  </div>
</div>
```

The pagination nav is, AFAIK, generated by Kaminari, and there's no way to customize that bit of markup without overriding Kaminari's default pagination helper, which seems out of scope for this PR.

I think we do need this `margin-bottom`-- because `.pagination--with-total` is a wrapper around `.pagination`, the outer div should have the same margin bottom in the end.